prov/usnic/src/usdf_domain.c

```diff
@@ -269,6 +274,12 @@ usdf_domain_open(struct fid_fabric *fabric, struct fi_info *info,
 	sin = info->src_addr;
 	if (info->src_addrlen != addrlen || sin->sin_family != AF_INET ||
 	    sin->sin_addr.s_addr != fp->fab_dev_attrs->uda_ipaddr_be) {
+		USDF_DBG_SYS(DOMAIN, "requested src_addr (%s) != fabric addr (%s)\n",
+			inet_ntop(AF_INET, &sin->sin_addr.s_addr,
+				requested, sizeof(requested)),
+			inet_ntop(AF_INET, &fp->fab_dev_attrs->uda_ipaddr_be,
+				actual, sizeof(actual)));
```

Comment:
I do note, however, that the if statement allows for `sin->sin_familty != AF_INET`, but you're unconditionally calling `inet_ntop()` with AF_INET.